core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockWorkspaceManagerDAO.scala

```diff
@@ -41,7 +39,8 @@ class MockWorkspaceManagerDAO(
   def mockCreateAzureCloudContextResult() = createCloudContextResult
   def mockInitialAzureRelayNamespaceResult() =
     MockWorkspaceManagerDAO.getCreateControlledAzureRelayNamespaceResult(StatusEnum.RUNNING)
-  def mockAzureRelayNamespaceResult() = createAzureRelayResult
+  def mockAzureRelayNamespaceResult() =
```

Comment:
I left in the `createAzureRelay` and `getCreateAzureRelayResult` methods in `WorkspaceManagerDAO` in case someone wants to create a relay from Rawls in the future. Do folks think I should instead delete them and their associated tests?

Reply:
I think going ahead with the deletion is the right thing to do-- it removes some of the test mocking. I have put it in a separate PR: https://github.com/broadinstitute/rawls/pull/2169.